elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -45,13 +73,32 @@ def render(element_html, data):
                 prefix = 'np.array('
                 suffix = ')'
 
+            # Mathematica reserved letters: C D E I K N O
+            mathematica_reserved = ['C', 'D', 'E', 'I', 'K', 'N', 'O']
+            if pl.inner_html(child) in mathematica_reserved:
+                mathematica_suffix = 'm'
+            else:
+                mathematica_suffix = ''
+
             # Create string for matlab and python format
-            matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=digits) + ';\n'
-            python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=digits) + suffix + '\n'
+            if pl.has_attrib(child, 'params-comment'):
+                matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + '; % ' + var_comment + '\n'
+                mathematica_data += pl.inner_html(child) + mathematica_suffix + ' = ' + pl.string_from_2darray(var_data, language='mathematica', digits=var_digits) + '; (* ' + var_comment + ' *)\n'
+                python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=var_digits) + suffix + ' # ' + var_comment + '\n'
+            else:
+                matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + ';\n'
+                mathematica_data += pl.inner_html(child) + mathematica_suffix + ' = ' + pl.string_from_2darray(var_data, language='mathematica', digits=var_digits) + ';\n'
+                python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=var_digits) + suffix + '\n'
```

Comment:
Can we avoid the code duplication here? Maybe first make a variable like `var_formatted_comment` which is either the empty string (if there is no comment) or something like `'(*' + comment + '*)'`.

Reply:
I did this, but again it has to be custom for each language, so I made `var_matlab_comment`, `var_mathematica_comment` and `var_python_comment`.

Not ideal but helps prevent the duplicate code?